CommonInstall.cmake

```diff
@@ -2,18 +2,19 @@
 
 # Uses a compatible subset of the install() syntax, but can install
 # files while preserving their relative directory, and additionally
-# installing them into the PROJECT_BINARY_DIR for compatibility with
-# superprojects. Files with an absolute filename are installed
-# directly into the DESTINATION, that is, their directory is not
-# preserved. If RELATIVE is set, directories are ignored on all files.
-
+# installing them into the PROJECT_BINARY_DIR or CMAKE_BINARY_DIR for
+# compatibility with superprojects. Files with an absolute filename
+# are installed directly into the DESTINATION, that is, their
+# directory is not preserved. If RELATIVE is set, directories are
+# ignored on all files.
+#
 # Usage: common_install(FILES <files> [COMPONENT <name>] [DESTINATION <prefix>]
-#                       [RELATIVE])
+#                      [RELATIVE] [CMAKE_BINARY_DIR])
```

Comment:
Done - reset to origin, cherry-pick first one and here we are. Something is seriously weird with my fork, since the CommonInstall was in both Eyescale and eile. hgmm.